pages/components/badge/badge.css

```diff
@@ -0,0 +1,76 @@
+@import "../../../styles/typography.css";
```

Comment:
can put comment of external import for sanity.

Reply:
Comments added.